src/test/cypress/support/pageobjects/multipleChoiceQuiz.ts

```diff
@@ -0,0 +1,19 @@
+import { POST } from '../constants';
+
+export class MultipleChoiceQuiz {
+    getQuizBody() {
+        return cy.get('.mc-question');
+    }
+
+    tickAnswerOption(optionNumber: number) {
+        this.getQuizBody()
+            .get('#answer-option-' + optionNumber + ' > .selection > .ng-fa-icon > .svg-inline--fa')
+            .click();
+    }
+
+    submit() {
+        cy.intercept(POST, '/api/exercises/*/submissions/live').as('createQuizExercise');
+        cy.get('.jhi-btn').contains('Submit').click();
```

Comment:
If the button has "Submit" as body you could use `.should('have.text', 'Submit')` instead. This does not trigger another DOM query, but only filters the buttons found by `get`

Reply:
Interesting didnt know that.